data/json/mutations/mutations.json

```diff
@@ -1045,6 +1045,16 @@
       }
     ]
   },
+  {
+    "type": "mutation",
+    "id": "TINYSTOMACH",
+    "name": { "str": "Tiny Stomach" },
+    "points": -3,
+    "description": "Either as the result of a lifesaving surgery, or as a congenital defect, your stomach is much smaller than usual. You'll have to eat your food in small servings during the day, rather than one or two large daily meals.",
```

Comment:
```suggestion
    "description": "Either as the result of a lifesaving surgery, or as a congenital defect, your stomach is much smaller than usual.  You'll have to eat your food in small servings during the day, rather than one or two large daily meals.",
```